ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/vlan-interfaces.md

```diff
@@ -185,9 +185,20 @@ interface Management1
 
 ##### ISIS
 
-| Interface | ISIS Instance | ISIS BFD | ISIS Metric | Mode |
-| --------- | ------------- | -------- | ----------- | ---- |
-| Vlan2002 | EVPN_UNDERLAY | True | - | - |
+| Interface | ISIS Instance | ISIS BFD | ISIS Metric | Mode | Level-1 Authentication Mode | Level-2 Authentication Mode |
+| --------- | ------------- | -------- | ----------- | ---- | --------------------------- | --------------------------- |
+| Vlan42 | EVPN_UNDERLAY | - | - | - | sha | sha |
+| Vlan83 | EVPN_UNDERLAY | - | - | - | md5 | md5 |
```

Comment:
am not sure if am missing something, in input you have given both as mode and in your logic you check if level-1/2 mode is defined then append that mode else - but here i can see that in all the example we have level-1 and level-2 mode.

Reply:
yes. If we choose `both` in inputs it is applicable for level_1 and level_2. You can seperately set level_1 and level_2 modes.

Reply:
Since I added all the interfaces level_1 and level_2 keys configured, both columns were filled. Now removed level_2 key on vlan interface 42. Showing `Level-2 authentication mode` column empty.